style.css

```diff
@@ -27,6 +27,10 @@
     flex: 1 1 auto;
   }
 
+  .top_meu {
+    display: none;
+  }
+
```

Comment:
- Kindly, your mobile version has overflow please be sure the mobile design still looks as expected to make your page fully responsive 👍 

https://github.com/rcniyigena/portfolio/assets/52010789/d8d85777-e8f8-4a53-a789-7c6955add2ea